dotcom-rendering/src/components/Kicker.tsx

```diff
@@ -1,43 +1,65 @@
 import { css } from '@emotion/react';
+import {
+	space,
+	textSans15,
+	textSansBold14,
+} from '@guardian/source/foundations';
+import { palette } from '../palette';
 import { Island } from './Island';
 import { PulsingDot } from './PulsingDot.importable';
 
-// Defines a prefix to be used with a headline (e.g. 'Live /')
 type Props = {
 	text: string;
 	color: string;
 	showPulsingDot?: boolean;
 	hideLineBreak?: boolean;
 };
 
-const kickerStyles = (colour: string) => css`
-	color: ${colour};
-	font-weight: 700;
-	margin-right: 4px;
+const textStyles = (color: string, showPulsingDot?: boolean) => {
+	return showPulsingDot
+		? css`
+				${textSansBold14}
+				color: ${palette('--kicker-text-live')};
+				background-color: ${palette('--kicker-background-live')};
+				width: fit-content;
+				padding: 0 ${space[1]}px;
+				display: flex;
+				flex-direction: row;
+				align-items: baseline;
+		  `
+		: css`
+				${textSans15}
+				color: ${color};
+				background-color: transparent;
+		  `;
+};
```

Comment:
This is not a blocker, but worth considering in terms of maintainability and code style alignment.

We tend to favour static styles that are composed in the `css` prop array. Furthermore, according to the [Emotion best practices, we should use the style prop for dynamic styles](https://emotion.sh/docs/best-practices#use-the-style-prop-for-dynamic-styles) 

```tsx
const liveStyles = css`color: palette('--kicker-text-live');`
const textStyles = css`background-color: transparent;`

const Kicker = ({ text, isLive, colour }: Props) => (
  <div
    style={{ color: colour }}
    css={[
      isLive ? liveStyles : textStyles,
    ]}
  >
    {props.text}
  </div>
)
```